docs/user_guide/models/amortizedlda.rst

```diff
@@ -0,0 +1,136 @@
+=============
+Amortized LDA
+=============
+
+**LDA** [#ref1]_ (Latent Dirichlet Allocation) posits a generative model where
+a set of latent topics generates collections of elements. In the case of single-cell RNA sequencing, we can think
+of these topics as gene modules and each cell as a collection of UMI counts. Other features that can be ascribed to these
+topics include surface proteins and accessible chromatin regions, all of which have discrete count values.
+This implementation (Python class :class:`~scvi.model.AmortizedLDA`) of LDA amortizes the
+cost of performing variational inference for each cell by training a common encoder. Note: this is not an exact implementation
+of the model described in the original LDA paper [#ref1]_.
+
+The advantages of amortized LDA are:
+- Can learn underlying topics without a reference.
+- Scalable to very large datasets (>1 million cells).
+
+The limitations of amortized LDA include:
+- Optimal selection of the number of topics is unclear.
+- Amortization gap in optimizing variational parameters.
+
+.. topic:: Tutorials:
+
+ - :doc:`/tutorials/notebooks/amortized_lda`
+
+
+Preliminaries
+==============
+Amortized LDA takes as input a cell-by-feature matrix :math:`X` with :math:`N` cells and :math:`F` features.
+Because the LDA model assumes the input is ordered, we refer to this format as the bag-of-words (BoW) representation
+of the feature counts.
+Additionally, the number of topics to model must be manually set by the user prior to fitting the model.
+
+
+Generative process
+==================
+
+Amortized LDA posits that the :math:`N` observed feature counts for cell :math:`c` are treated as ordered. For the :math:`n`th feature count,
+the feature observed, :math:`x_{cn}` is produced according to the following generative process:
+
+.. math::
+   :nowrap:
+
+   \begin{align}
+    \text{For each topic $k$:} \\
```

Comment:
Looks a bit weird rendered, would be better not in the latex env, same with the other for loop